app/lib/features/space/pages/chats_page.dart

```diff
@@ -22,6 +25,7 @@ class SpaceChatsPage extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final chats = ref.watch(relatedChatsProvider(spaceIdOrAlias));
     final related = ref.watch(spaceRelationsOverviewProvider(spaceIdOrAlias));
+    final size = MediaQuery.of(context).size;
```

Comment:
here we do not pass the context form around ... seems weird that we do in the other case.

Reply:
I think I used ```shellContext``` here for the determination of modal. I will do that also in other place.